modules/ocf_www/files/vhost-web.jinja

```diff
@@ -36,16 +36,14 @@
             Options +ExecCGI
             AllowOverride None
             Require all granted
-            SetHandler fastcgi-script
+            SetHandler fcgid-script
         </Directory>
 
-        <FilesMatch "\.ph(p3?|tml)$">
+        <FilesMatch "\.ph(p[3457]?|tml)$">
```

Comment:
why is six afraid of seven? because seven eight nine

this joke doesn't really fit but i couldn't think of a better one to comment on the lack of a php 6

:partycat:

Reply:
Haha, that actually fits pretty well, let's just see if there will be a PHP 9 or not, maybe it was eaten? ;)

:partycat: actually shows up on my phone but not on desktop, thanks GitHub :(